src/convertToObject.js

```diff
@@ -3,16 +3,30 @@
 /**
  * Implement convertToObject function:
  *
- * Function takes a string with styles (see an example in [stylesString.js](./stylesString.js))
+ * Function takes a string with styles
+ * (see an example in [stylesString.js](./stylesString.js))
  * and returns an object where CSS properties are keys
- * and values are the values of related CSS properties (see an exampl in [test file](./convertToObject.test.js))
+ * and values are the values of related CSS properties
+ * (see an exampl in [test file](./convertToObject.test.js))
  *
  * @param {string} sourceString
  *
  * @return {object}
  */
 function convertToObject(sourceString) {
-  // write your code here
+  return sourceString
+    .split(';')
+    .reduce((resultOfReduce, element) => {
+      const [key, value] = element
+        .split(':')
+        .map(partOfElement => partOfElement.trim());
+
+      if (key.length) {
```

Comment:
```suggestion
      if (key) {
```

Reply:
I guess, It will also work, because:
<img width="251" alt="image" src="https://github.com/mate-academy/js_style-to-object/assets/86789161/b55b47b9-e08f-4c85-88a6-9cf0664dee15">

Reply:
done